monitoring/uss_qualifier/webapp/docker-compose-webapp.yaml

```diff
@@ -0,0 +1,48 @@
+version: '3.9'
+services:
+  mock-rid:
+    extends:
+      file: ../../uss_qualifier/scripts/docker-compose_qualifier_mocks.yaml
+      service: mock
+  mock-scdsc:
+    build:
+      context: ../../
+      dockerfile: mock_uss/Dockerfile
+    command: ../mock_uss/run_locally_scdsc.sh
```

Comment:
`scdsc` mock is coming up with this service configuration.
Should I create a seperate docker-compose file for above mock or combine with `uss_qualifier/scripts/docker-compose_qualifier_mocks.yaml`?